app/assets/stylesheets/dummy/index.scss

```diff
@@ -0,0 +1,54 @@
+/*Puedes agregar un archivo scss segun tu criterio para facilitar la busqueda o que no se acopla a ninguno de los ya existentes */
+/* General */
+@charset 'utf-8';
+
+// Base
+/* Aqui se podria declarar los estilos basicos per de manera general para la app*/
+@import 'base/_reset';
```

Comment:
Good, just get used to add comments in English xD.

You can also import multiple files like this:
```
@import 'base/_reset',
                'base/_typography',
                'base/_base'...
```

Also, it's not needed to write the underscore `_` ... you can do it like:

```
@import `base/reset`, 
               `base/typography`, ...
```